cmd/siklu/siklu_nfs_boot.c

```diff
@@ -27,11 +27,10 @@ void setup_bootargs(const char *bootargs) {
 	}
 	mtdparts = env_get(ENV_MTDPARTS);
 	if (! mtdparts) {
-		mtdparts = CONFIG_SIKLU_DEFAULT_MTD_PARTS;
-		SK_LOG_NFS("Using default mtdparts: %s\n", mtdparts);
+		const char *mtdids;
+		board_mtdparts_default(&mtdids, &mtdparts);
```

Comment:
Where is `mtdids` used?

Reply:
Now I see that `board_mtdparts_default()` is called from generic code.

Reply:
Added comments in this file and in board/freescale/mx6ullevk/mx6ullevk.c to clarify.

Reply:
Added comments in this file and in board/freescale/mx6ullevk/mx6ullevk.c to clarify.